### DISCUSSION TITLE
本地MP4视频 FFmpegDemuxer错误

### DISCUSSION THREAD

**Discussion by shenzq518** [ZuodaoTech/everyone-can-use-english#985] [General]
DEMUXER_ERROR_COULD_NOT_OPEN: FFmpegDemuxer: open context failed
解码波形失败: enjoy://library/videos/1fe8b68c9b9107c0947a93935590f5d5.mp4

**Comment by an-lee** [comment]
有尝试过删掉重新添加吗？

**Comment by shenzq518** [comment]
导入后直接点击进行语音转文字会提示该错误 但是过一会再重试的话就正常了
